bin/debug/export_single_user.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import json
+import logging
+from logging import config
+import argparse
+import numpy as np
+import uuid
+
+import emission.pipeline.export_stage as epe
+import emission.core.wrapper.user as ecwu
+
+if __name__ == '__main__':
```

Comment:
Is this supposed to be the new "pulled out" separate pipeline? How do you envisage that this is run?
The current intake pipeline does not take any arguments by design since it is launched automatically through a cronjob. If we launch this other script with a cronjob as well, what arguments would we pass in?

Reply:
As discussed, this is simply a testing/launch infrastructure that I implemented to be able to run the separate pipeline. It is parallel in style and usage to the bin/debug/intake_single_user.py file. 

Question: Would you like this removed from the commit? It was mostly for myself/running the actual script.

Reply:
Yes, I would like you to remove this since it doesn't seem to add additional functionality beyond the existing script.